tools/common/Frameworks.cs

```diff
@@ -524,15 +524,13 @@ public static void Gather (Application app, AssemblyDefinition product_assembly,
 	// This checks if a framework is unavailable due to bugs in Xcode (such as Apple forgetting to ship a library or headers for a framework, which seems to happen at least once a year).
 	public static bool IsFrameworkBroken (Application app, string framework)
 	{
-		if (app.Platform == ApplePlatform.WatchOS && app.IsSimulatorBuild && Driver.XcodeProductVersion == "12A6159" /* Xcode 12 beta 1 */) {
+		if (app.Platform == ApplePlatform.WatchOS && app.IsSimulatorBuild && Driver.XcodeProductVersion == "12A6163b" /* Xcode 12 beta 12 */) {
```

Comment:
We'll have a lot of work this summer if Apple releases 12 betas 😄

Reply:
lol, every two weeks and a late/fall release... maybe it will :)